README.md

```diff
@@ -272,10 +269,43 @@ const response = await replicate.predictions.create(options);
   "metrics": {},
   "created_at": "2022-04-26T22:13:06.224088Z",
   "started_at": null,
-  "completed_at": null
+  "completed_at": null,
+  "urls": {
+    "get": "https://api.replicate.com/v1/predictions/ufawqhfynnddngldkgtslldrkq",
+    "cancel": "https://api.replicate.com/v1/predictions/ufawqhfynnddngldkgtslldrkq/cancel",
+    "stream": "https://streaming.api.replicate.com/v1/predictions/ufawqhfynnddngldkgtslldrkq"
```

Comment:
I suggest commenting this (since we're using `jsonc`) so it's not surprizing that it's missing most of the time.

```suggestion
    "stream": "https://streaming.api.replicate.com/v1/predictions/ufawqhfynnddngldkgtslldrkq" // Only present if `options.stream` is `true`
```